src/resource/jsonschema/__tests__/map.test.js

```diff
@@ -19,7 +19,7 @@ describe('mapSchema', () => {
                 additionalProperty: 'additionalProperty',
                 value: 'string',
             };
-            const mapping = mapSchema(object, schema, schema.registry, isDefined);
+            const mapping = mapSchema(object, schema, isDefined);
```

Comment:
We essentially wrap a resource/schema in a `Reference`, so we no longer need a `registry`; we can just use `reference.resource.schema`